src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-clone-modal/user-clone-modal.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="modal-header">
```

Comment:
Please use the Modal component here.

Reply:
Just saw we have a few modals that are not using the ModalComponent, so I have created https://tracker.ceph.com/issues/33606.